exporter/prometheusexporter/prometheus_test.go

```diff
@@ -21,6 +21,11 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
+	"github.com/open-telemetry/opentelemetry-service/factories"
```

Comment:
same nit: otel import shoul be grouped on the bottom

Reply:
I am going to look for a linter that can verify this.